@types/contentful-html-rich-text-converter/package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
yeah you shouldn't need that? You should just add a type declaration file

Reply:
I thought I needed because looking at the other types folders, they all have package.json files
 
<img width="285" alt="Screen Shot 2022-11-15 at 15 06 44" src="https://user-images.githubusercontent.com/16595804/201994084-583bbc50-21c5-4f87-bd3a-644b1dfa13eb.png">

I can delete it, but it wouldn't follow the pattern of the other folders. Do you think it's better to remove this file?